content/features/featuresDeepDive/materials/using/masterPBR.md

```diff
@@ -96,6 +96,49 @@ Once the conversion done, let's see the custom options available on this version
 - **useMicroSurfaceFromReflectivityMapAlpha**: the reflectivity texture contains the microSurface or glossiness information in its alpha channel.
 - **useAmbientInGrayScale**: the ambient occlusion is forced to read only from the red channel of the ambient texture or from the red channel of the metallic texture.
 
+## Controlling Specular Reflections on Metallic-Roughness Materials
+There are times when having control over the power or color for specular reflections of a PBR metallic-roughness material is desirable. Further, there are substances that can’t be accurately rendered without this type of control. Typically, a PBR specular-glossiness material is used for these types of substances, but there are still ways to use a metallic-roughness material to render a substance which needs modification to its specular reflections. 
+
+For glTF models, there is a Khronos extension called `KHR_materials_specular` which adds `specular` and `specularColor` parameters to the metallic-roughness material. Additionally, the extension allows using textures to control these values through `specularTexture` and `specularColorTexture`. These parameters affect the dielectric BRDF to change the specular power or color of F0 reflections, previously only attainable using specular-glossiness materials. If a glTF file is authored with this extension, the Babylon.js loaders will translate the glTF material description to a `PBRMaterial` with the parameters set correctly.
+
+Babylon.js materials constructed as metallic-roughness can also exert control over specular highlights through several parameters on `PBRMaterial`. Beyond the basic setup of the material, these parameters need to be defined to shape specular reflections.
+
+- **metallicF0Factor**: This is the same as `specularFactor` from `KHR_materials_specular`. This parameter is multiplied with `metallicReflectanceColor` to control the intensity of specular reflections.
+- **metallicReflectanceColor**: This is the same as `specularColorFactor` from `KHR_materials_specular`. This parameter sets specular color at F0. As surface normals move toward F90, this value is interpolated to white.

```

Comment:
For maximum clarity, do we need to explain what F0 and F90 are, or do we expect the user to know what they mean?

Also, `metallicReflectanceColor` is the color at F90, not F0.

`As surface normals move toward F90, this value is interpolated to white` => it is interpolated to white by default (because 1 is the default value for `metallicF0Factor`), but in the general case it is interpolated to (`metallicF0Factor`, `metallicF0Factor`, `metallicF0Factor`).

Reply:
Yes, we should probably define F0 and F90. I didn't take the time because all other documentation around specular on our site and Khronos's repo assume the reader knows what they are. But we should take some time to explain. I will add that. 

For metallicReflectanceColor, I went back and forth on this. We say in the API it's color at F90, but the definition in the [specular extension](https://github.com/KhronosGroup/glTF/tree/main/extensions/2.0/Khronos/KHR_materials_specular#extending-materials) is "The F0 color of the specular reflection (linear RGB)." We say that the equivalent in Babylon to specularColorFactor is metallicReflectanceColor but our definitions are opposite one another. So I went to a test in the sandbox and if I set metallicReflectacnceColor to red, only the F0 specular color shows red and the F90 values are white. I am trying to strike a balance between the definitions and what people see. 

My struggle here is what you are saying makes complete sense when you look at the code, but not everyone (including me) will open the engine code and look so I am trying to find a balance between what the engine does and what the user perceives. Do you have a better way to resolve the opposite definitions between the Khronos extension and our docs?

Reply:
@Popov72, I added an opening paragraph explaining F0 and F90. Let me know what you think of that. @sebavan, what do you think about the comment above?

Reply:
yup good catch !!! let s change the code doc as well then

Reply:
I fixed the doc:

https://github.com/BabylonJS/Babylon.js/pull/14067